app/assets/stylesheets/styles/components/_footer.css.scss

```diff
@@ -0,0 +1,32 @@
+
+footer {
+  height: $footer-height;
+  background: RGB(32, 25, 23);
```

Comment:
Name of function `RGB` should be written in all lowercase letters with hyphens instead of underscores